tasks.py

```diff
@@ -750,7 +750,7 @@ def unittest(
         command += " --buffer"
     if verbose:
         command += " --verbosity 2"
-    if parallel:
+    if parallel or label == "nautobot":
```

Comment:
Can we do anything to auto-select a `parallel_workers` value that doesn't overly tax the system? I've found when running locally that just `--parallel` defaults to a number of workers (8 on my Mac) that results in 100% CPU consumption while tests are running, which tends to make the rest of the system pretty unhappy, so I usually use `--parallel-workers=6` so as to leave some overhead free for the system. Obviously in CI we'd want to use the default for optimal runtime, but in the local dev case I think it'd be desirable to throttle it somewhat.

Reply:
In fairness this may be a self-inflicted wound, I need to change my Docker resource allocation so that it's not allowed to use all my CPUs at once. :-)